GDJS/GDJS/Extensions/Builtin/MouseExtension.cpp

```diff
@@ -61,6 +61,9 @@ MouseExtension::MouseExtension() {
   GetAllExpressions()["LastEndedTouchId"].SetFunctionName(
       "gdjs.evtTools.input.getLastEndedTouchId");
 
+  GetAllExpressions()["MouseWheelDelta"].SetFunctionName(
+      "gdjs.evtTools.input.getMouseWheelDelta");
+
```

Comment:
Also add here the bindings from conditions to JS functions for the new conditions.